### PR TITLE
Async loading of some scripts and CSS to speed up initial load

### PR DESCRIPTION
**Pull request by canadianveggie**
Tested it locally by turning on Slow 3G network simulation.

This should improve our Lighthouse scores.

Addresses this Lighthouse feedback:
Screen Shot 2022-02-20 at 10 39 05 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1031,4 +1060,5 @@ <h2>Sharing theme</h2>
       };
     }
   </script>
+  <script src="public/dictionary.js" async></script>
```
Comment (wachunga) [inline index.html:1063]: 👍 

I would like to investigate using [script type="module"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) (which implicitly defer) so we don't have globals floating around but I believe it's not trivial with Alpine

File: index.html [hunk 2]
```diff
@@ -1008,6 +1029,14 @@ <h2>Sharing theme</h2>
             return defaultValue;
           }
         },
+        logEvent(eventType, eventProperties) {
+          try {
+            amplitude.getInstance().logEvent(eventType, eventProperties);
+          } catch (error) {
```
Comment (wachunga) [inline index.html:1035]: A more robust approach would be to queue them until it's available but probably not necessary given these won't happen on page load

**Review by wachunga — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (canadianveggie) [reply index.html:1035]: We might want an initial event, but the first event we have right now is `level_start` and that doesn't happen until the first guess is made.

Comment (canadianveggie) [reply index.html:1035]: And in my testing amplitude actually loaded very quickly. I wasn't even able to trigger this catch with `Slow 3G` simulation on.

**Pull request merged**